js-basic/Callback/callback.js

```diff
@@ -0,0 +1,8 @@
+function fun() {
+  console.log('the exam is done ');
+}
+function fun2(fun) {
+  console.log('result is out ');
+  fun();
+}
```

Comment:
Naming conventions should be  proper and informative.

Reply:
all variable names are change into informative